File: scratch/mark_polish_done.php
<?php
require_once dirname(__DIR__) . '/config.php';
try {
    $conn = connectDB();
    // Marca o polonês como done
    $stmt = $conn->prepare("UPDATE odysee_publish_queue SET status = 'done', error_message = 'Manual override' WHERE language_id = 10 AND status IN ('pending', 'processing')");
    $stmt->execute();
    echo "Atualizado: " . $stmt->rowCount() . " linha(s).\n";
} catch (Exception $e) {
    echo "Erro: " . $e->getMessage();
}

File: scratch/mark_polish_done.py
import sys
sys.path.append('/app')
from worker import get_db_connection
c = get_db_connection()
cursor = c.cursor()
cursor.execute("UPDATE odysee_publish_queue SET status = 'done', error_message = 'Cancelado manualmente pois video ja postado' WHERE language_id = 10 AND status IN ('pending', 'processing')")
print(f"Atualizado: {cursor.rowcount}")
c.commit()
c.close()
